rustsec/tests/support/example_advisory_v4_from_ghsa.md

```diff
@@ -0,0 +1,24 @@
+```toml
+id = "RUSTSEC-2001-2101"
+package = "base"
+date = "2001-02-03"
+url = "https://www.youtube.com/watch?v=jQE66WA2s-A"
+categories = ["code-execution", "privilege-escalation"]
+keywords = ["how", "are", "you", "gentlemen"]
+aliases = ["CVE-2001-2101"]
+cvss = "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:H/A:H"
+syndicated_from = "github-advisory-database"
```

Comment:
This should be an array if we use this as we have multiple origins and we should use the well known abbreviations e.g. GHSA, NVD etc. to keep it ergonomic - compact and memorable.